pkg/crc/network/httpproxy/proxy.go

```diff
@@ -260,3 +264,16 @@ func HTTPTransport() http.RoundTripper {
 
 	return proxyConfig.HTTPTransport()
 }
+
+func RevalidateProxyCAFile() error {
+	proxyConfig, err := NewProxyConfig()
+	if err != nil {
+		return err
+	}
+	if proxyConfig.ProxyCAFile != "" {
+		if proxyConfig.ProxyCACert == "" {
```

Comment:
The 2 commits before this one ignore various errors happening in `readProxyCAData`, this check for `ProxyCACert` is an indirect way of checking if we ignored any errors.
We also ignore invalid proxy settings from the environment, but this will not be detected here (not necessarily mandatory, but would be nice if everything could be reported to the user).

The commit logs are short on details about the bugs that are being fixed, so I'd need to debug this again to see if there are easy alternatives I can suggest to improve this :(